lib/chai/utils/addChainableMethod.js

```diff
@@ -79,7 +78,7 @@ module.exports = function (ctx, name, method, chainingBehavior) {
 
         var assert = function assert() {
           var old_ssfi = flag(this, 'ssfi');
-          if (old_ssfi && config.includeStack === false)
+          if (old_ssfi)
```

Comment:
That flag is already checked in the assert() method and checking it here has no real benefit aside from making the logic more complicated